src/containers/MyNdla/components/UserAvatar.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import { breakpoints, misc, mq } from '@ndla/core';
+// import EditProfilePicture from '../MyProfile/components/EditProfilePicture';
+import Avatar from './Avatar';
+
+type UserAvatarProps = {
+  userName: string | undefined;
+  userId: string | undefined;
+  showProfileButton?: boolean;
+};
+
+const UserAvatarContainer = styled.div`
+  position: relative;
+`;
+
+const DesktopButtonContainer = styled.div`
+  ${mq.range({ until: breakpoints.tablet })} {
+    display: none;
+  }
+  ${mq.range({ from: breakpoints.tablet })} {
+    position: absolute;
```

Comment:
Mest av nysgjerrighet, men hvorfor må denne ha position: absolute?

Reply:
for å plassere knappen. du må ha parent relativ og child absolute og da er det enklere å få denne plassert i forhold til avatar bildet. hvertfall sånn jeg pleier å gjøre det